test/CMakeLists.txt

```diff
@@ -1572,3 +1572,33 @@ add_custom_test(test_regression_opencl_float_mi100 VEGA_DISABLED HIP_DISABLED GF
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  512, 28, 28 --weights 128, 512, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
     COMMAND	${ENVS_REGRESSION_ISSUE_1012} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --cmode conv --pmode default --group-count 1 --input 64,  64,  56, 56 --weights 256, 64,  1, 1 --pads_strides_dilations 0 0 1 1 1 1 ${ARGS_REGRESSION_ISSUE_1012}
 )
+
+
+set(CONV_CK_IGEMM_FWD_V4R4R4_XDLOPS_NHWC_ENV
+    MIOPEN_FIND_MODE=normal
+    MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvCkIgemmFwdV4r4r4XdlopsNhwc)
+
+add_custom_test(test_conv_ck_igemm_fwd_v4r4r4_xdlops_nhwc FLOAT_ENABLED HALF_ENABLED  GFX908_ENABLED SKIP_UNLESS_ALL
```

Comment:
The new solver is applicable for gfx90a. Add `GFX90A_ENABLED`

Reply:
done